### DISCUSSION TITLE
Simconnect data from FS

### DISCUSSION THREAD

**Discussion by egpd1701** [nguyenquyhy/Flight-Tracker-StreamDeck#320] [Q&A]
Hi. I was trying to get the pressure readings through the simconnect client your code does successfully get the pressure in mb. I was needing to get the pressure in Hg as well, directly reading from the sim using the simconnect variables and the "inHg" as a unit instead of doing the math on it. It seems to read or at least start to read the variable then stops. I cannot get it to read anything past the decimal. e.g. 29.92 is reading as 29. So I thought I change the variable unit from int to float64 but all that did was give me the wrong figures and mess up the rest of the vars. I was wondering if you were seeing the same issue or if it's something that I need to do in the car definition?

**Comment by nguyenquyhy** [comment]
Put inHg as unit and 2 as decimals.

**Reply by egpd1701** [reply]
Hi. Can I ask where to put the 2 as the 29 was the data coming directly from simconnect.flightstatus before it wrote to any program var.
